presentation/presentation_test.go

```diff
@@ -26,3 +26,19 @@ func TestRemoveChoicesWithPics(t *testing.T) {
 		}
 	}
 }
+
+func TestExtractText(t *testing.T) {
+	testFilePath := "./testdata/extraction.pptx"
+
+	want := "TestSubtestHeader 1Test ErrewrTests sf s. dsf Sdfsd xvxv Header 2Qweqre wefs dfSd f sdf"
```

Comment:
Is it possible to get those by page/slide number? I.e. extract page/slide text?
Seems like the concept of pages/slides probably exists in spreadsheets

Reply:
@gunnsth I almost did this, but one thing. Should we enumerate slides starting from 0 or 1?

Reply:
Left is starting from 0 for now